app/services/terms_of_use/constants.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module TermsOfUse
+  module Constants
+    PROVISIONER_COOKIE_NAME = 'CERNER_CONSENT'
+    PROVISIONER_COOKIE_VALUE = 'ACCEPTED'
+    PROVISIONER_COOKIE_PATH = '/'
+    PROVISIONER_COOKIE_EXPIRATION = 2.minutes
```

Comment:
Just curious.. should the cookie ever expire? 👀

Reply:
yea, otherwise if someone declines on another page then sites wont know

Reply:
Okay.